spec/controllers/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe BookSuggestionController, type: :controller do
+
+  describe 'POST #create' do
```

Comment:
Here you should consider if exists a case where the api returns a different status than 200

Reply:
Done